### PR TITLE
#260: Read default options from Info.plist

### PR DESCRIPTION
**Pull request by bonkey**
#260: Read default options from Info.plist

### CODE REVIEW

**Comment by bonkey** [comment]
@hani-ibrahim I dropped your changes in `APIClientErrorsSpec` as there were a bit too many and not really important IMHO differences to keep it.

File: AtlasSDK/AtlasSDK/Errors.swift [hunk 1]
```diff
@@ -20,6 +20,7 @@ public enum AtlasConfigurationError: AtlasErrorType {
 
     case incorrectConfigServiceResponse
     case missingClientId
+    @available( *, deprecated, message = "Sales channel should be taken from config service, when ready")
```
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/Errors.swift]: Should not anymore

**Comment by hani-ibrahim** [comment]
👍

File: AtlasSDK/AtlasSDK/Errors.swift (hunk 1 above)
Comment (bonkey) [inline AtlasSDK/AtlasSDK/Errors.swift]: Yes, fixing here or other PR

**Comment by bonkey** [comment]
👍

File: AtlasSDK/AtlasSDK/Errors.swift (hunk 1 above)
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/Errors.swift]: I would say other PR and fix these whole country/sales channel thing 
and close all these PRs now :D

Comment (bonkey) [inline AtlasSDK/AtlasSDK/Errors.swift]: 💯

**Pull request merged**